src/code_translation/ir/nodes.py

```diff
@@ -51,13 +51,12 @@ def sql_string(self) -> str:
     def to_code(self) -> str:
         return f'read_sql("{self.sql_string}", SOME CON)'
```

Comment:
What is this actually used for? Does this need a reference to the sql_access_method also?

Reply:
the methods to_code were now competely replaced by to_cst_translation. So we can delete them.

Reply:
> the methods to_code were now competely replaced by to_cst_translation. So we can delete them.

Okok, will do :+1: